src/main/java/core/basesyntax/service/ReportToCsv.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax.service;
+
+import core.basesyntax.db.StockDao;
+import core.basesyntax.db.StockDaoStorageImpl;
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class ReportToCsv implements Report {
+    private static final String FILE_NAME = "report.csv";

```

Comment:
Avoid hardcoding the file name directly in the class. Consider passing the file name as a parameter to the constructor or using a configuration file to manage such constants, as suggested in the checklist.